cpp/src/arrow_chunk_writer.cc

```diff
@@ -90,7 +90,42 @@ Result<std::shared_ptr<arrow::Table>> ExecutePlanAndCollectAsTable(
 
 // implementations for VertexPropertyChunkWriter
 
```

Comment:
maybe to add a comment to indicate the validate method for which operation in .cc?

Reply:
sovled